clients/devops_challenge_component/src/styles.css

```
@@ -0,0 +1,5 @@
+@import 'tailwindcss';
+
+@config '../tailwind.config.js';
+
+@import '../../tailwind-base.css';
```

Stylelint (17.9.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)